x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/create_trusted_app_flyout.tsx

```diff
@@ -197,7 +216,7 @@ export const CreateTrustedAppFlyout = memo<CreateTrustedAppFlyoutProps>(
               id="xpack.securitySolution.trustedapps.createTrustedAppFlyout.expiredLicenseMessage"
               defaultMessage="Your Kibana license has been downgraded. Future policy configurations will now be globally assigned to all policies. For more information, see our "
             />
-            <EuiLink external href={`${docLinks.links.securitySolution.trustedApps}`}>
+            <EuiLink target="_blank" href={`${docLinks.links.securitySolution.trustedApps}`}>
```

Comment:
It needs to open docs in a new tab, `external` prop only adds the icon.

Reply:
Yeah on new browsers, the `target="_blank"` means implicit `rel="noopener, noreferrer"`, but still might need to add the `rel` attribute here depending on which browser version we support.